README.md

```diff
@@ -37,9 +37,13 @@ Magma has three major components
 
 ![Magma architecture diagram](docs/readmes/assets/magma_overview.png?raw=true "Magma Architecture")
 
-## Documentation
+## Documentation Guide
```

Comment:
I would either leave this title as 'Documentation' or change it to something like 'Useful Resources'. Adding the word 'Guide' reads confusing to me.

Reply:
Fixed.